xml/System.Workflow.Runtime.Hosting/SqlWorkflowPersistenceService.xml

```diff
@@ -524,7 +524,7 @@
           <format type="text/markdown"><![CDATA[  
```

Comment:
The link in line 522 is also broken, but the text is redundant. Could you remove it, so that the return value description just reads, "\<see langword="true" /> indicates that the batch should be committed."?

Reply:
Done.